db/fixtures/notification_types.yml

```diff
@@ -254,6 +254,16 @@
   :expires_in: 24.hours
   :level: :error
   :audience: global
+- :name: cloud_tenant_delete_success
+  :message: 'Deleting Cloud Tenant %{subject} completed successfully'
```

Comment:
I'd recommend adding a "." at the end of this line so that the punctuation matches other messages, but otherwise LGTM.

Reply:
Ok, fixed